app/components/hamburger/hamburger.tsx

```diff
@@ -0,0 +1,159 @@
+"use client";
+import React, { useState } from "react";
+import { useSwitchChain } from "@starknet-react/core";
+import { Check, LibraryBig, Menu, X } from "lucide-react";
+
+interface HamburgerProps {
+  theme: "dark" | "light";
+}
+
+const STARKNET_CHAINID = {
+  SN_MAIN: "0x534e5f4d41494e", // Mainnet
+  SN_SEPOLIA: "0x534e5f5345504f4c4941", // Sepolia testnet
+};
+
+const CustomSelect: React.FC<{ theme: "dark" | "light" }> = ({ theme }) => {
```

Comment:
Add dark mode for drop-down

Reply:
<img width="1512" alt="Screenshot 2024-10-01 at 12 18 30" src="https://github.com/user-attachments/assets/125af1bf-d402-47fb-9ef4-3aab903d00f4">


please, there's no dark mode

Reply:
Just use the dark background colors for it, please.

Reply:
<img width="1512" alt="Screenshot 2024-10-03 at 13 16 40" src="https://github.com/user-attachments/assets/274dc6d3-9aa9-4458-8b99-cbed3ff6c13e">


done